libcloud/storage/drivers/s3.py

```diff
@@ -1029,7 +1056,9 @@ def __init__(self, key, secret=None, secure=True, host=None, port=None,
         self.name = 'OUTSCALE Ceph RGW S3 (%s)' % (region)
         self.ex_location_name = region
         self.region_name = region
-        self.connectionCls = S3RGWOutscaleConnection
+        signature_version = kwargs.get('signature_version', 'aws2')
+        self.connectionCls =\
```

Comment:
Here is an example of how we handle `region` connection class argument in the same manner - https://github.com/apache/libcloud/blob/trunk/libcloud/compute/drivers/rackspace.py#L142